packages/pn-personafisica-webapp/src/redux/notification/reducers.ts

```diff
@@ -147,7 +147,6 @@ const notificationSlice = createSlice({
 
       if (paymentInfo?.pagoPa) {
         state.paymentsData.pagoPaF24 = [
-          ...state.paymentsData.pagoPaF24,
```

Comment:
Removing this line doesn't  fix the problem in case of multiple payments. Try login using user 'cesare' and search for notification having iun 'UDEZ-QHVE-UPXY-202408-V-1'. Selecting one of the two payments available and mocking the response to cause an error will cause the other payment to disappear. You should change the code in a way that the payments not involved are not removed from the store and only the affected payment is updated using the proper status and detail info.